_foundations/eclipse.md

```diff
@@ -24,9 +24,8 @@ core:
   legalName: "Eclipse Foundation, Inc."
   location: "US"
   nonprofit: "501C6"
-  softwareType: "devtool"
-  licenses: "EPL-1.0"
-  copyrightPolicy: "CLA"
+  licenses: "EPL-2.0"
+  copyrightPolicy: "https://www.eclipse.org/legal/ECA.php"
```

Comment:
Design is that copyrightPolicy is an enum CLA|DCO|Assign, but will consider adding the URL separately.

Reply:
DCO then.